Assets/Scripts/MIDI2Event/Midi2Event.cs

```diff
@@ -72,6 +72,24 @@ public void Update(double deltaTime)
             }
         }
 
+        //returns the current number of seconds per beat
+        public double SecPerBeat()
+        {
+            return _usPerQuarter * US_TO_S;
+        }
+
+        //returns the current number of beats per second
+        public double BeatPerSec()
+        {
+            return 1 / SecPerBeat();
+        }
+
+        //returns true if this system is currently playing
+        public bool IsPlaying()
+        {
+            return _isPlaying;
+        }
```

Comment:
These could probably be Properties? (Up to you whether you make this change, since it is personal preference and the line is kind of debatable, but I feel like this may be a case where the calculation is small enough that it feels more like a Property kind of thing than the longer kind of calculation or action that would be better suited to a normal method)

Reply:
changed!